Packs/ANYRUN/Integrations/ANYRUN/ANYRUN.py

```diff
@@ -653,277 +1035,220 @@ def contents_from_history(filter, response):
+            new_task['Related'] = f'[{related}]({related})'
+            reformatted_contents.append(new_task)
+        command_results = CommandResults(
+            outputs_prefix='ANYRUN.Task',
+            outputs_key_field='ID',
+            outputs=formatted_contents,
+            readable_output=tableToMarkdown(
+                f'Task History - Filtered By "{filter}"' if filter else 'Task History',
+                reformatted_contents
+            )
+        )
     else:
-        human_readable = 'No results found.'
-        entry_context = None
-
-    return_outputs(readable_output=human_readable, outputs=entry_context, raw_response=response)
-
-
-def get_report(task_id):
-    """Make API call to ANYRUN to get task report
-
-    Parameters
-    ----------
-    task_id : str
-        The unique task ID of the analysis whose report to fetch.
-
-    Returns
-    -------
-    dict
-        Response JSON from ANYRUN API call.
-    """
-
-    try:
-        # according to the any-run documentation, this request should work:
-        # https://any.run/api-documentation/#api-Analysis-GetReport
-        url_suffix = f'analysis/{task_id}'
-        response = http_request('GET', url_suffix=url_suffix)
-    except DemistoException as exc:
-        if exc.res and exc.res.status_code != 403:
-            raise
-
-        # in case of 403, try a work-around suggested by customer
-        url_suffix = 'analysis/'
-        params = {
-            'task': task_id,
-        }
-        response = http_request('GET', url_suffix=url_suffix, params=params)
+        command_results = CommandResults(
+            outputs=None,
+            readable_output='No results found'
+        )
 
-    return response
+    return command_results
 
 
-def get_report_command():
-    """Return ANYRUN analysis report to Demisto"""
-    args = demisto.args()
+def get_report_command(client: Client, args: Dict):
+    """Return ANYRUN analysis report to XSOAR"""
     task_id = args.get('task')
-    response = get_report(task_id)
+    response = client.get_report(str(task_id))
 
-    images = images_from_report(response)
+    images = client.get_images_from_report(response)
     contents = contents_from_report(response)
-    formatting_funcs = [underscore_to_camel_case, make_capital, make_singular, make_upper]
-    formatted_contents = travel_object(contents, key_functions=formatting_funcs)
-
-    dbot_score = generate_dbotscore(response)
-    entity = ec_entity(response)
-
-    entry_context = {
-        'ANYRUN.Task(val.ID && val.ID === obj.ID)': {
-            'ID': task_id,
-            **formatted_contents
-        },
-        **dbot_score,
-        **entity
+    reports = response.get('data', {}).get('analysis', {}).get('reports')
+    contents['ID'] = task_id
+    formatted_contents = travel_object(contents)
+    formatted_contents['Reports'] = reports
+
+    dbot_scores: List = generate_dbotscore(response)
```

Comment:
The name `dbot_scores`/`generate_dbotscore` doesn't informative enough (You return List of CommandResult).

Reply:
Fixing